Makefile

```diff
@@ -20,14 +13,13 @@ build/manifest.json: node_modules/.uptodate
 clean:
 	find . -type f -name "*.py[co]" -delete
 	find . -type d -name "__pycache__" -delete
-	rm -f node_modules/.uptodate .pydeps
+	rm -f node_modules/.uptodate
 	rm -rf build
 
 ## Run the development H server locally
 .PHONY: dev
-dev: build/manifest.json .pydeps
-	@bin/hypothesis --dev init
-	@bin/hypothesis devserver
+dev: build/manifest.json
+	tox -e dev
```

Comment:
`make dev` isn't _just_ a wrapper for `tox -e dev`: it still installs the JavaScript dependencies.